packages/core/svelte/package.json

```diff
@@ -24,12 +24,14 @@
         "@solana/wallet-adapter-base": "^0.6.0",
         "@solana/wallet-adapter-wallets": "^0.10.1",
         "@solana/web3.js": "^1.29.2",
+        "eventemitter3": "^4.0.7",
```

Comment:
`@solana/wallet-adapter-base` exports the `EventEmitter` type and class, you may not need it as a dep. It's also not clear to me where this being used.

Reply:
This prevents an issue that only arises on local development in SSR mode.

The error comes from the transpiled `adapter.js`

```javascript
import EventEmitter from 'eventemitter3';
export { EventEmitter };
export class BaseWalletAdapter extends EventEmitter {
}
export var WalletAdapterNetwork;
(function (WalletAdapterNetwork) {
    WalletAdapterNetwork["Mainnet"] = "mainnet-beta";
    WalletAdapterNetwork["Testnet"] = "testnet";
    WalletAdapterNetwork["Devnet"] = "devnet";
})(WalletAdapterNetwork || (WalletAdapterNetwork = {}));
//# sourceMappingURL=adapter.js.map
```

The import to `eventemitter3` doesn't exist in the bundle (probably because it's not actually used).

The error was something like `cannot extend value of undefined`, so we decided to include it as dep so other uses running `SvelteKit` in local development wouldn't run into this issue.

Reply:
@jordansexton this solves the way to call the `walletStore`. If you remember in the first PR, I was importing the store inside an `onMount()` method because in SSR the adapter was crashing. Including `eventemitter3` the issue is solved and the app doesn't give any errors in development and in production.